resources/definitions/matterhacker_pulsexe.def.json

```
@@ -0,0 +1,231 @@
+{
+    "version":2,
+    "name":"Matterhackers Pulse XE",
+    "inherits":"fdmprinter",
+    "metadata":{
+        "author":"Chris Inacio <nacho319@inacio.net>",
+        "visible":true,
+        "manufacturer":"Matterhackers",
+        "file_formats":"text/x-gcode",
+        "first_start_actions":["MachineSettingsAction"],
+        "machine_extruder_trains":{
+            "0":"matterhackers_e3dlite_extruder",
+            "1":"matterhackers_e3dv6_extruder"
+        }
+    },
+	"overrides":
+	{
+	    "machine_name":{
+	        "default_value":"Matterhackers Pulse XE"
+	    },
+	    "machine_width":{ "default_value":250 },
```

```suggestion
	    "machine_width": { "default_value": 250 },
```
Please use spaces after : This makes it much easier to maintain in the future. I've not marked all the cases where this needs to be changed.